### DISCUSSION TITLE
Money mask with comma as decial separator does not work properly when reciving data from model binding

### DISCUSSION THREAD

**Discussion by pelmered** [alpinejs/alpine#4222] [7. Bugs (archived)] [answered]
I reported this issue a while back on the Filament repository, but from what understand the issue is not in Filament, but in Alpine.
Here is a link to the Filament issue: https://github.com/filamentphp/filament/issues/10867

The issue is when you use comma as a decimal separator, like we do in most of Europe for example. 

See the following code:
```
    <div x-data="{ money: 12345.67 }">
        Alpine: <input x-mask:dynamic="$money($input, ',', ' ')" x-model="money">
    </div>
```
This should give an input with the value `12345,67`, but instead we get `1 234 567`

Also see this demo that @danharrin created in that Filament issue: https://wirebox.app/b/47r9n

I'm not sure, but this might be a duplicate of: https://github.com/alpinejs/alpine/discussions/3009
If so, here are at least code example and better steps for reproducing.

**Comment by SimoTod** [comment, marked as answer]
The $money plugin is not that powerful. It's not going to do the conversion in your behalf and it will also save back the formatted value.

If you want to use a comma, the original value must be a string that uses a comma as decimal separator.

See https://wirebox.app/b/x6d7d

**Reply by pelmered** [reply]
Ah, got it. Thank you!
